### PR TITLE
changes so far found in v7.1.0-alpha.2 vs v7.0.2

### PR DESCRIPTION
**Pull request by y3rsh**
# Use snapshots to compare protocol analysis across builds

## Bugs found

- <https://opentrons.atlassian.net/browse/RQA-2001> 
  - This is manual blowout? Should not throw error?
- <https://opentrons.atlassian.net/browse/RQA-2032> 
  - addressed with #14145 

## Signoff

- [ ] All these changes are expected and safe

### CODE REVIEW

**Comment by sfoster1** [comment]
@y3rsh it may be tough to do in this pr obviously but since git diff tools work on lines, can we format this json with like one key/value pair per line?

File: app-testing/tests/__snapshots__/analysis_test/test_analysis_json[OT2_P10S_P300M_TC1_TM_MM_2_11_Swiftanalysis].json [hunk 1]
```diff
@@ -2671,7 +2671,7 @@
           ],
           "metadata": {
             "displayCategory": "tipRack",
-            "displayName": "Opentrons 96 Tip Rack 20 µL",
+            "displayName": "Opentrons OT-2 96 Tip Rack 20 µL",
```
Comment (y3rsh) [inline app-testing/tests/__snapshots__/analysis_test/test_analysis_json[OT2_P10S_P300M_TC1_TM_MM_2_11_Swiftanalysis].json:2674]: Expected change due to  change in shared-data
[`shared-data/labware/definitions/2/opentrons_96_tiprack_300ul/1.json`](https://github.com/Opentrons/opentrons/blob/2f0b36d0504b1353cfb541e86d4046b0252ecb57/shared-data/labware/definitions/2/opentrons_96_tiprack_300ul/1.json)

File: app-testing/tests/__snapshots__/analysis_test/test_analysis_json[OT2_P20S_P300M_HS_6_1_HS_WithCollision_Erroranalysis].json [hunk 2]
```diff
@@ -50,7 +50,7 @@
               "dropOffset": {
                 "x": 0.0,
                 "y": 0.0,
-                "z": 0.5
+                "z": 1.0
```
Comment (y3rsh) [inline app-testing/tests/__snapshots__/analysis_test/test_analysis_json[OT2_P20S_P300M_HS_6_1_HS_WithCollision_Erroranalysis].json:53]: @sanni-t please take a look at this and all the other changes to values in this PR when you can 😊

File: app-testing/tests/__snapshots__/analysis_test/test_analysis_json[OT2_P300M_P20S_MM_TM_TC1_5_2_6_PD40Erroranalysis].json [hunk 3]
```diff
@@ -6881,7 +6881,7 @@
             "errorInfo": {
               "args": "('Cannot aspirate more than pipette max volume',)",
               "class": "AssertionError",
-              "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/commands/publisher.py\", line 113, in publish_context\n    yield\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/instrument_context.py\", line 238, in aspirate\n    self._core.aspirate(\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/core/legacy_simulator/legacy_instrument_core.py\", line 112, in aspirate\n    new_volume <= self._pipette_dict[\"working_volume\"]\n"
+              "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/commands/publisher.py\", line 113, in publish_context\n    yield\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/instrument_context.py\", line 267, in aspirate\n    self._core.aspirate(\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/core/legacy_simulator/legacy_instrument_core.py\", line 119, in aspirate\n    new_volume <= self._pipette_dict[\"working_volume\"]\n"
```
Comment (y3rsh) [inline app-testing/tests/__snapshots__/analysis_test/test_analysis_json[OT2_P300M_P20S_MM_TM_TC1_5_2_6_PD40Erroranalysis].json:6884]: Think I will ignore `traceback` key.

**Pull request closed without merging**